src/main/java/hw_2/Task2.java

```diff
@@ -0,0 +1,12 @@
+package hw_2;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+        //ver 1
+        System.out.printf("%d %s%s",x+y,x,y);
+        //ver 2
+        //System.out.println(x+y+" "+x+""+y);
+    }
```

Comment:
Could you add spaces between after and before operators. I mean not "a+b" but "a + b". Could you change it in all places please

Reply:
Fixed